src/components/notebook/RichOutput.tsx

```diff
@@ -1,125 +1,138 @@
-import React, { useState } from 'react'
-import ReactMarkdown from 'react-markdown'
-import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
-import { oneLight } from 'react-syntax-highlighter/dist/esm/styles/prism'
-import { StreamOutputData } from '@runt/schema'
-import { FilePlus, Edit, ChevronDown, Info, ZoomIn, X } from 'lucide-react'
-import { AnsiStreamOutput } from './AnsiOutput.js'
-import './RichOutput.css'
+import React, { useState } from "react";
+import ReactMarkdown from "react-markdown";
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
+import { oneLight } from "react-syntax-highlighter/dist/esm/styles/prism";
+import { StreamOutputData } from "@runt/schema";
+import { FilePlus, Edit, ChevronDown, Info, ZoomIn, X } from "lucide-react";
+import { AnsiStreamOutput } from "./AnsiOutput.js";
+import "./RichOutput.css";
```

Comment:
Hmmmmm... am I have formatting changes between machines? I should probably just standardize us on `deno fmt` so it's the same as `runtimed/runt`.

Reply:
Well now I've forced `deno fmt` but... that also involved changing all the damn files.